src/main.go

```diff
@@ -2,22 +2,93 @@ package main
 
 import "github.com/gin-gonic/gin"
 import "net/http"
+import "github.com/jmoiron/sqlx"
+import _ "github.com/lib/pq"
 
 const NameParam = "name"
 
+var db *sqlx.DB
+var router *gin.Engine
+
 func main() {
-	r := gin.Default()
-	r.GET("/hello", func(c *gin.Context) {
+	initDatabase()
+	initAPI()
+
+	router.Run()
+}
+
+func initDatabase() {
+	db = sqlx.MustConnect("postgres", "postgres://ormkzpwm:x6uyb8EA034nb8rP8YMgy7udyYrYUuVw@manny.db.elephantsql.com:5432/ormkzpwm")
+}
+
+func initAPI() {
+	router = gin.Default()
+	router.GET("/hello", func(c *gin.Context) {
 		c.JSON(http.StatusOK, gin.H{
 			"message": "Hello world!",
 		})
 	})
-	r.GET("/greet/:" + NameParam, func(c *gin.Context) {
+	router.GET("/greet/:"+NameParam, func(c *gin.Context) {
 		name := c.Param(NameParam)
 		message := "Hello " + name + "!"
 		c.JSON(http.StatusOK, gin.H{
 			"message": message,
 		})
 	})
-	r.Run() // listen and serve on 0.0.0.0:8080 (for windows "localhost:8080")
+	router.GET("/patient/all", GetAllPatients)
+	router.GET("/location/all", GetAllLocations)
+	router.GET("/hospital/all", GetAllHospitals)
+}
+
+func GetAllPatients(c *gin.Context) {
+	patientRows, _ := db.Queryx("select name, illness, birth_date, location_id from patient")
```

Comment:
Can you get all the attributes from `patient`. `select * from patient`. Also, please change to `db.Select()`, it'll be easier to read. https://jmoiron.github.io/sqlx/#getAndSelect